.github/workflows/deploy.yml

```diff
@@ -49,18 +49,27 @@ jobs:
           kubectl config set-context ${KUBE_CLUSTER} --cluster=${KUBE_CLUSTER} --user=deploy-user --namespace=${KUBE_NAMESPACE}
           kubectl config use-context ${KUBE_CLUSTER}
 
+      - name: Fetch and set SHARED_IP_RANGES_LAA environment variable
```

Comment:
Instead of repeating this, what do you think of sticking in its own job and then using the output of that to set SHARED_IP_RANGES_LAA?

Reply:
Hiya, we repeat the other jobs so i think keeping this in may be the best approach to avoid a cumbersome workflow. What ideas did you have in mind?